themes/src/main/resources-community/theme/base/admin/messages/admin-messages_pt_BR.properties

```diff
@@ -421,19 +421,19 @@ import-users=Importar usuários
 import-groups=Importar grupos
 import-clients=Importar clientes
 import-identity-providers=Importar provedores de identificação
-import-realm-roles=Importar roles do realm
-import-client-roles=Importar roles de cliente
+import-realm-roles=Importar fun\u00E7\u00F5es do realm
+import-client-roles=Importar fun\u00E7\u00F5es de cliente
```

Comment:
When comparing these changes with the ones from lines 413-414, there's a discrepancy in how we use capital letters for realm and "cliente". This is actually found throughout the whole file. We should pick one flavor and make it consistent too.

Reply:
Good point. I did some updates to those as well. Seems to me most of the time lower case client is better (I left a few exceptions - either due to punctuation or because consistency isn't required).